engine/src/main/java/org/terasology/rendering/openvrprovider/OSValidator.java

```diff
@@ -0,0 +1,61 @@
+package org.terasology.rendering.openvrprovider;
+
+/* Mainly used to locate the OpenVR library on different platforms. */
+public class OSValidator {
```

Comment:
This looks like the kind of class that shouldn't be deep down in the render. @msteiger or @Cervator can perhaps provide a better home package for it?